bea-missed-schedule.php

```diff
@@ -1,42 +1,59 @@
 <?php
-
 /*
-  Plugin Name: BEA Missed Schedule
-  Plugin URI: http://www.beapi.fr
-  Description: Fix <code>Missed Schedule</code> Future Posts Cron Job: find missed schedule posts that match this problem every 1 minute and it republish them correctly fixed 10 items per session.
-  Author: BeAPI
-  Author URI: http://www.beapi.fr
-  Version: 0.1
- */
+Plugin Name: BEA Missed Schedule
+Plugin URI: http://www.beapi.fr
+Description: Fix <code>Missed Schedule</code> Future Posts Cron Job: find missed schedule posts that match this problem every 1 minute and it republish them correctly fixed 10 items per session.
+Author: Be API
+Author URI: https://beapi.fr
+Version: 0.2
+*/
 
 class BEA_Missed_Schedule {
-	const delay = 1; // in minutes
-	const option_name = 'bea_missed_schedule';
+	/**
+	 * Delay in minutes
+	 */
+	const DELAY = 1;
 
+	/**
+	 * BEA_Missed_Schedule constructor.
+	 */
 	public function __construct() {
 		add_action( 'init', array( __CLASS__, 'init' ), 0 );
 	}
 
+	/**
+	 * Delete option when deactive this plugin
+	 */
 	public static function deactivation() {
-		delete_option( self::option_name );
+		delete_option( 'bea_missed_schedule' );
 	}
 
+	/**
+	 * Init an option with as value the expiration delay in seconds, if delay is over, exec the cleanup function
+	 *
+	 * @return bool
+	 */
 	public static function init() {
-		$last = get_option( self::option_name, false );
-		if ( ( false !== $last ) && ( $last > ( time() - ( self::delay * 60 ) ) ) ) {
+		$last = get_option( 'bea_missed_schedule', false );
+		if ( ( false !== $last ) && ( $last > ( time() - ( self::DELAY * 60 ) ) ) ) {
 			return false;
 		}
 
-		update_option( self::option_name, time() );
+		update_option( 'bea_missed_schedule', time() );
 
 		return self::publish_missed_schedule();
 	}
 
+	/**
+	 * The cleanup function that publish posts if WP missed the schedule
+	 *
+	 * @return bool
+	 */
 	public static function publish_missed_schedule() {
 		global $wpdb;
 
 		$scheduled_ids = $wpdb->get_col( "SELECT `ID` FROM `{$wpdb->posts}` WHERE( ((`post_date`>0)&&(`post_date`<=CURRENT_TIMESTAMP())) OR ((`post_date_gmt`>0)&&(`post_date_gmt`<=UTC_TIMESTAMP())) )AND `post_status` = 'future' LIMIT 0, 10" );
-		if ( false == $scheduled_ids ) {
+		if ( false === $scheduled_ids || empty( $scheduled_ids ) ) {
 			return false;
```

Comment:
If no return true, just return will be enough.

Reply:
if there is no `return true` at the end of the method, then the `return false` isn't quite necessary. just `return` will do the job.